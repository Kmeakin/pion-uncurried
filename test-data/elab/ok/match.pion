let _ = match false {
    x => true,
};