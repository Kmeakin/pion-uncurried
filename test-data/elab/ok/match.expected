let _ : Bool = match false {
x => true};
